setup.py

```diff
@@ -19,8 +19,20 @@
 from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 from version import version
+from pkg_resources import parse_version
+
+# Toil version check -- Raise warning instead of using intall_requires to avoid virtualenv conflicts
+toil_minor_version = '3.1'
+next_minor_version = toil_minor_version[:2] + str(int(toil_minor_version[3]) + 1)
```

Comment:
Use parse_version. It exists so you don't have to hand-code brittle version parsing like that.